services/dynamodb/src/test/resources/log4j.properties

```diff
@@ -31,5 +31,4 @@ log4j.logger.org.apache.http.wire=DEBUG
 #log4j.logger.org.apache.http=DEBUG
 #log4j.logger.org.apache.http.wire=WARN
 
-#log4j.logger.com.amazonaws=DEBUG
-log4j.logger.com.amazonaws.services.s3.internal.Mimetypes=WARN
+#log4j.logger.software.amazon.awssdk=DEBUG
```

Comment:
Needed?

Reply:
It's not needed and will help debugging. 
The removed lines are obsolete anyway.